README.md

```diff
@@ -77,7 +77,7 @@ The modularized code-base approach provides few benefits:
 
 ### Module types and module dependencies
 
-This is a diagram present dependencies between project modules (Gradle sub-projects).
+This diagram present dependencies between project modules (Gradle sub-projects).
```

Comment:
This should probably be "presents" instead of "present".

Reply:
Done. Thank you.